bentoml/handlers/dataframe_handler.py

```diff
@@ -193,6 +193,7 @@ def __init__(
         typ="frame",
         input_dtypes=None,
         is_batch_input=True,
+        cors="*",
```

Comment:
could you also add the parameter to the docstring above?

Reply:
Added to the docstring. I searched for it on the `__init__` method and didn't notice it's on the class definition.